### ISSUE TITLE
Update for 'September 18, 2015' API changes

### ISSUE THREAD

**Issue by justdmitry** [justdmitry/NetTelegramBotApi#21]
Telegram team made some changes in API, marked as **September 18, 2015**: 
- Bots can now download files and media sent by users.
- Added `getFile` and `File`.

Source: https://core.telegram.org/bots/api-changelog#september-18-2015

**Comment by justdmitry** [comment]
Sample code for downloading file/image data:

```
var photos = update.Message.Photo;
if (photos != null)
{
  var webClient = new WebClient();
  foreach(var photo in photos)
  {
      Console.WriteLine("  New image arrived: size {1}x{2} px, {3} bytes, id: {0}", photo.FileId, photo.Height, photo.Width, photo.FileSize);
      var file = bot.MakeRequestAsync(new GetFile(photo.FileId)).Result;
      var tempFileName = System.IO.Path.GetTempFileName();
      webClient.DownloadFile(file.FileDownloadUrl, tempFileName);
      Console.WriteLine("    Saved to {0}", tempFileName);
  }
}
```

For other file types - use `FileId` from corresponding `Message` property

**Comment by justdmitry** [comment]
Implemented in version **3.7.50926**
